playbooks/templates/tor-blocklists-update.sh.j2

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# Exit on first error
+set -e
+
+# Tor Project list
+# Check timestamp (if present)
+# If timestamp doesn't exist or is old enough
+# - Download exit node list 
+# - Load IPs to ipset
+# - Create/Update timestamp
+
+now=$(date +%s)
+past=$((now - {{ block_tor_exit_nodes_lists.do_not_redownload_within_secs }}))
+timestamp_file=/tmp/{{ block_tor_exit_nodes_lists.torproject.filename }}-timestamp
+url={{ block_tor_exit_nodes_lists.torproject.url }}
+file=/tmp/{{ block_tor_exit_nodes_lists.torproject.filename }}
+ipset={{ block_tor_exit_nodes_lists.torproject.ipset_ipv4 }}
+
+if [[ ! -f $timestamp_file || $(cat $timestamp_file) -lt $past ]]; then
+    echo "Downloading torproject exitnodelist"
+    curl -sSL $url -o $file
+
+    echo "Loading torproject exitnodelist to ipset"
+    cat $file | xargs -n 1 ipset -exist add $ipset 
+
+    echo "Creating/Updating torproject timestamp"
+    echo "$(date +%s)" > $timestamp_file
+else
+    echo "It's too soon to redownload torproject exitnodelist"
```

Comment:
This might misinform users that the tor list is not being used. How about something like this?

```suggestion
    echo "It's too soon to redownload torproject exitnodelist, using cached version from $(cat $timestamp_file)."
```